commands/call.go

```diff
@@ -1,62 +1,30 @@
 package commands
 
 import (
-	"os"
+	"fmt"
 
-	"github.com/fnproject/cli/client"
-	"github.com/fnproject/cli/common"
-	"github.com/fnproject/cli/objects/route"
-	"github.com/fnproject/cli/run"
-	fnclient "github.com/fnproject/fn_go/client"
 	"github.com/fnproject/fn_go/provider"
 	"github.com/urfave/cli"
 )
 
 type callCmd struct {
 	provider provider.Provider
-	client   *fnclient.Fn
 }
 
 // CallCommand returns call cli.command
 func CallCommand() cli.Command {
 	cl := callCmd{}
 	return cli.Command{
-		Name:    "call",
-		Usage:   "\tInvoke a remote function",
-		Aliases: []string{"cl"},
-		Before: func(c *cli.Context) error {
-			var err error
-			cl.provider, err = client.CurrentProvider()
-			if err != nil {
-				return err
-			}
-			cl.client = cl.provider.APIClient()
-			return nil
-		},
-		ArgsUsage:   "<app-name> </path>",
-		Flags:       route.CallFnFlags,
+		Name:        "call",
+		Usage:       "\tPrompts for migration.",
+		Aliases:     []string{"cl"},
+		ArgsUsage:   "",
 		Category:    "DEVELOPMENT COMMANDS",
-		Description: "This command calls explicitly a function.",
+		Description: "This command no longer executes a function, instead it prompts users to migrate their routes to fn/triggers.",
```

Comment:
maybe be nice to see 'you should use invoke' or link to details about the migration here

Reply:
nvm, it does do that. just not here i guess